steps/terraform.yaml

```diff
@@ -148,7 +148,7 @@ steps:
       command: apply
       workingDirectory: '$(System.DefaultWorkingDirectory)/$(buildRepoSuffix)/components/${{ parameters.component }}'
       environmentServiceName: ${{ parameters.serviceConnection }}
-      commandOptions: "${{ parameters.applyCommandOptions }} -auto-approve tfplan-${{ parameters.environment }}-${{ lower(replace(parameters.location, ' ', '')) }}-${{ parameters.component }}"
```

Comment:
This was added because terraform specifically requires the plan to be specified at the end, otherwise process fails, see below:
- https://www.terraform.io/cli/commands/apply
- https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=266374&view=logs&j=b9e95c09-f529-583e-447a-411b3ebcb77d&t=54625de3-d704-5019-9da4-e7653b66a850&l=16